src/main/java/com/shift4/response/PaymentMethod.java

```diff
@@ -135,4 +152,93 @@ private void set(String name, Object value) {
             other.put(name, value);
         }
     }
+
+    public static class ThreeDSecure {
+        private String currency;
+        private Integer amount;
+
+        @JsonIgnore
+        private final Map<String, Object> other = new HashMap<>();
+
+        public String getCurrency() {
+            return currency;
+        }
+
+        public Integer getAmount() {
+            return amount;
+        }
+
+        public String get(String name) {
+            return Shift4Utils.toStringNullSafe(other.get(name));
+        }
+
+        @JsonAnySetter
+        private void set(String name, Object value) {
+            other.put(name, value);
+        }
+    }
+
+
+    public static class GooglePay {
+        private String cardBrand;
+        private String cardType;
+        private String first6;
+        private String last4;
+
+        @JsonIgnore
+        private final Map<String, Object> other = new HashMap<>();
+
+        public CardBrand getCardBrand() {
+            return CardBrand.fromValue(cardBrand);
+        }
+
+        public String getCardBrandAsString() {
+            return cardBrand;
+        }
+
+        public CardType getCardType() {
+            return CardType.fromValue(cardType);
+        }
+
+        public String getCardTypeAsString() {
+            return cardType;
+        }
+
+        public String getFirst6() {
+            return first6;
+        }
+
+        public String getLast4() {
+            return last4;
+        }
+
+        public String get(String name) {
+            return Shift4Utils.toStringNullSafe(other.get(name));
+        }
+
+        @JsonAnySetter
+        private void set(String name, Object value) {
+            other.put(name, value);
+        }
+    }
+
+    public static class PaymentMethodFlow {
+        private FlowNextAction nextAction;
```

Comment:
this should be string and treated the same way as other response enums (getX and getXAsString). Enum on field does not allow to ready newly introduced values without SDK update.